File: ProjetoPOO/Models/Pet.cs
using System;

namespace ProjetoPOO.Models
{
    class Pet : ModeloId
    {
        public string Nome { get; set; }
        public string Especie { get; set; }
        public string Raca { get; set; }
        public int IdCliente { get; set; }

        public Pet() { }

        public Pet(int id, string nome, string especie, string raca, int idCliente)
        {
            Id = id;
            Nome = nome;
            Especie = especie;
            Raca = raca;
            IdCliente = idCliente;
        }

        public override string ToString()
        {
            return $"ID: {Id} | Nome: {Nome} | Espécie: {Especie} | Raça: {Raca} | ID Cliente: {IdCliente}";
        }
    }
}

File: ProjetoPOOPython/Models/pet.py

from Models.modeloId import ModeloId


class Pet(ModeloId):
    def __init__(self, id: int = 0, nome: str = "", especie: str = "", raca: str = ""):
        super().__init__(id)
        self.nome = nome
        self.especie = especie
        self.raca = raca

    def __str__(self):
        return f"ID: {self.id} | Nome: {self.nome} | Espécie: {self.especie} | Raça: {self.raca}"
